pybleau/app/ui/plot_template_manager_view.py

```diff
@@ -0,0 +1,90 @@
+from pyface.confirmation_dialog import confirm
+from pyface.constant import YES
+from traits.api import HasStrictTraits, Instance, List, Str, Button, Bool, \
+    Property, Any
+from traitsui.api import ListStrEditor, VGroup, HGroup, Spring, Readonly, \
+    Item, View
+
+from pybleau.app.model.plot_template_manager import PlotTemplateManager
+from pybleau.app.model.tests.test_plot_template_manager import \
+    FakePlotTemplateInteractor
+
+
+class PlotTemplateManagerView(HasStrictTraits):
+    """ Pop-up dialog for viewing and deleting available plot templates
+    """
+    # View model
+    model = Instance(PlotTemplateManager)
+
+    # Templates in the list that are selected
+    selected_plot_templates = List(Str)
+
+    # Delete button - to delete the selected template(s)
+    delete_from_templates = Button("Delete Selected")
+
+    #: Rescan button - to refresh the list of templates
+    rescan_for_templates = Button("Rescan Template Directory")
+
+    #: Bool for whether a template (or templates) is selected
+    is_selected = Property(Bool(False), depends_on="selected_plot_templates")
+
+    #: View class to use. Modify to customize.
+    view_klass = Any(View)
+
+    #: Is the UI interactive? Set to False for testing
+    interactive = Bool(True)
+
+    def traits_view(self):
+        plot_template_edit = ListStrEditor(selected="selected_plot_templates",
+                                           multi_select=True, editable=True,
```

Comment:
`editable=True`: does that mean that the ui supports renaming an existing template? It might make users think it does but I don't see how the implementation enables it.

Reply:
Good catch. It doesn't seem like they are editable, even with this argument set to `True`. But I have removed it nonetheless.

Reply:
I think that supporting template renaming would be good to add as a separate ticket.

Reply:
Added a new issue "Allow users to rename plot templates" (#123)

Reply:
Issue #123